DigitalLearningSolutions.Web/Controllers/Support/SupportTicketsController.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.Support
+{
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Helpers;
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Support;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.FeatureManagement;
+
+    [SetDlsSubApplication]
+    [SetSelectedTab(nameof(NavMenuTab.Support))]
+    public class SupportTicketsController : Controller
+    {
+        private readonly IConfiguration configuration;
+        private readonly IFeatureManager featureManager;
+
+        public SupportTicketsController(IFeatureManager featureManager, IConfiguration configuration)
+        {
+            this.featureManager = featureManager;
+            this.configuration = configuration;
+        }
+
+        [Route("/{dlsSubApplication}/Support/Tickets")]
+        [Authorize(Policy = CustomPolicies.UserCentreAdminOrFrameworksAdmin)]

```

Comment:
We typically have these on the Controller if possible, is there a reason not to do so here?

Reply:
Good spot, its hang over from the main support controller. I've moved them up.